openedx/core/lib/call_stack_manager/core.py

```diff
@@ -0,0 +1,163 @@
+"""
+Get call stacks of Model Class
+in three cases-
+1. QuerySet API
+2. save()
+3. delete()
+
+classes:
+CallStackManager -  stores all stacks in global dictionary and logs
+CallStackMixin - used for Model save(), and delete() method
```

Comment:
You should delete this file, now that you've moved `call_stack_manager` over to `djangoapps`.